ui/assets/icons/CollapseAll.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+const CollapseAllIcon = ({ width = 24, height = 24, fill = "currentColor", style = {} }) => (
```

Comment:
will you also parallelly put these icons in sistent too

Reply:
@sudhanshutech that is such a great request and such a terrible one both at the same time. Why?